### ISSUE TITLE
..< ligature don't seem to work (Monaspace Argon Var, Monaspace Neon Var, maybe others)

### ISSUE THREAD

**Issue by neilenns** [githubnext/monaspace#10]
I'm using VSCode on Windows 11 and have the following in my settings.json file:

```javascript
  "editor.fontLigatures": "'calt', 'liga', 'dlig', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08'",
```

The .- and .= ligatures work but the ..< doesn't:

<img width="86" alt="image" src="https://github.com/githubnext/monaspace/assets/9524118/be3d4664-1308-4cb6-b1fe-eb1198b9e459">

I tried with both "Monaspace Argon Var" and "Monaspace Neon Var" as the font family.

**Comment by drbenvincent** [comment]
Try `Monaspace Argon` i.e. without the var

**Comment by neilenns** [comment]
I tried this on a second machine and it worked fine (settings for vscode sync'ed by settings sync, I just installed the fonts on the other machine) 🤔

**Comment by Zeitgeist117** [comment]
On Linux. ligatures does not seem to work on alacritty, simple terminal(patched), emacs or neovim with either the otf fonts or the variable ttf fonts. just to double check ligatures works fine on fonts like Fira Code and CaskaydiaCove nerd font on my system

**Comment by M1ndo** [comment]
> On Linux. ligatures does not seem to work on alacritty, simple terminal(patched), emacs or neovim with either the otf fonts or the variable ttf fonts. just to double check ligatures works fine on fonts like Fira Code and CaskaydiaCove nerd font on my system

Yes indeed, tried all fonts of monspace on Emacs no ligatures worked , i also tried a patched version of Monaspace Neon Nerd Font didn't work either . Other fonts work fine.

**Comment by Zeta611** [comment]
Hey, is there *any* working terminal on macOS that supports Monaspace ligatures? I've never seen it work except for VSCode with
```
  "editor.fontLigatures": "'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'calt', 'liga', 'dlig'",
```
config.
I've tried iTerm2, Kitty, and Warp but none of them seem to show the ligatures for Monaspace.

**Comment by idan** [comment]
Fixed in ca8015b
